apps/are_form/behaviours/referencelist.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const staticExclusionDates = require('../lib/staticExclusionDates');
+const staticAppealStages = require('../lib/staticAppealStages');
+const moment = require('moment');
+const dateFormat = 'dddd DD MMMM YYYY';
+
+module.exports = superclass => class ReferenceList extends superclass {
+    getValues(req, res, callback) {
+        super.getValues(req, res, (err, values) => {
+            let json = req.sessionModel.toJSON();
+
+            if (req.url === '/appealstages') {
+                json['reference-appeal-list'] = [].concat(staticAppealStages.getstaticAppealStages());
+                json['reference-appeal-list-count'] = json['reference-appeal-list'].length;
+            } else if (req.url === '/exclusiondates') {
+                json['reference-exclusiondate-list-england-and-wales'] =
+                    [].concat(staticExclusionDates.getExclusionDays('England & Wales'));
```

Comment:
where is the `England & Wales` being set? it's dangerous to have something like this because of the spacing and `&`

Reply:
Being used in staticExclusionDates.js, which is referencing the json['country-of-hearing'] in result.js from the sessionModel, pointing to id="country-of-hearing" in first-page.html.

Looks like the name is coming from the field names on the first page.

Reply:
keep for now because too much work to refactor